styles/GameRow.module copy.css

```diff
@@ -0,0 +1,64 @@
+#listContainer {
+	min-height: 100vh;
+	padding: 0 0.5rem;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+#nameContainer {
+	padding: 20px;
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
```

Comment:
```suggestion
	align-items: center;
	flex-wrap: wrap;
```

hopefully prevent names from flowing off-screen in the x direction.

![image](https://user-images.githubusercontent.com/20099646/110237278-053bfb80-7ef0-11eb-8d07-e841ca71c035.png)